actors/builtin/miner/policy.go

```diff
@@ -28,8 +32,8 @@ func init() {
 	}
 }
 
-// The maximum number of sectors that a miner can have simultaneously active.
-// This also bounds the number of faults that can be declared, etc.
+// Filecoin Parameter: The maximum number of sectors that a miner can have (and implicitly the maximum number of faults, recoveries, etc.) 
+// Motivation: This guarantees that actors operations per miner are bounded. Also, this reflects the limit of sectors that can be proven in a day via WindowPoSt.
 // TODO raise this number, carefully
 // https://github.com/filecoin-project/specs-actors/issues/470
 const SectorsMax = 32 << 20 // PARAM_FINISH
```

Comment:
**Change request:** `SectorsMax` is specific to a particular `RegisteredProof`. For example, NSE will have a much smaller `SectorsMax`. Why? One of the constrains that sets the `SectorsMax` is the number of sectors that can be proven in 24 hours via a WindowPost. With NSE, the number of sectors that can be proven in a WindowPoSt is smaller than SDR, hence the SectorsMax chosen for SDR won't work for NSE WindowPoSt.

Reply:
Now tracked on #470

Reply:
This is however a requirement from proof as we found in the Params Audit (added more motivation here and planning to add more in the PR): https://github.com/filecoin-project/specs-actors/issues/470#issuecomment-660933651

Reply:
Continuing conversation on #470.

Reply:
I take this one back - this has been clarified. I will remove the second part of the sentence.